guides/source/active_record_multiple_databases.md

```diff
@@ -53,21 +53,21 @@ Let's add a second database called animals and replicas for both databases as we
 this, we need to change our `database.yml` from a 2-tier
 to a 3-tier config.
 
-If a primary configuration is provided, it will be used as the "default" configuration. If
-there is no configuration named `"primary"`, Rails will use the first configuration as default
+If a "default" configuration is provided, it will be used by default. If
+there is no configuration named `"default"`, Rails will use the first configuration as default
```

Comment:
I agree this documentation is kind of confusing but the changes here aren't correct. `primary` refers to the name of the configuration object. So in the case of the config seen here it would be `env_name: :production, name: :primary`. `default` when used in the context of connections is the default shard name used. The `name` and the `shard` are two distinct concepts so I'd generally not recommend naming the shard as the same name as the config. In your application you ended up with that because you generated the shard names, but IMO it should be done more like the following:


```ruby
connects_to shards: {
  default: { writing: :primary },
  shard_two: { writing: :secondary }
}
```

My example above uses different names to demonstrate the 2 different concepts. `shards` and `roles` are the context in which a connection is looked up from the handler. But `name` and `env` are part of the configuration. Does that make sense? It's kind of convoluted, I'm trying to make it simpler over time.

Reply:
got it! okay let's change it so

Reply:
okay, got it, thanks!